config/jobs/kubernetes/sig-network/sig-network-kind.yaml

```diff
@@ -625,3 +625,56 @@ periodics:
     testgrid-tab-name: sig-network-kind, multizone
     description: Runs tests against a Multizone Kubernetes in Docker cluster
     testgrid-alert-email: antonio.ojea.garcia@gmail.com, robertjscott@google.com
+# network test against kubernetes master branch with `kind` and kube-proxy detectLocalMode="NodeCIDR"
+- interval: 6h
+  name: ci-kubernetes-kind-network-detect-local-node-cidr
```

Comment:
This is a copy of "ci-kubernetes-kind-network" except for the changes noted below.